core/math/quaternion.cpp

```diff
@@ -313,6 +313,41 @@ Quaternion::Quaternion(const Vector3 &p_axis, real_t p_angle) {
 	}
 }
 
+static bool try_set_perpendicular_to(const real_t &in_x, const real_t &in_y, const real_t &in_z, real_t &out_x, real_t &out_y, real_t &out_z) noexcept {
```

Comment:
```suggestion
static bool _try_set_perpendicular_to(real_t p_in_x, real_t p_in_y, real_t p_in_z, real_t &r_out_x, real_t &r_out_y, real_t &r_out_z) {
```
Some format and standard details